zetaclient/zetacore/client.go

```diff
@@ -6,7 +6,8 @@ import (
 	"strings"
 	"sync"
 
-	rpchttp "github.com/cometbft/cometbft/rpc/client/http"
+	cometbft_rpc_client "github.com/cometbft/cometbft/rpc/client"
```

Comment:
```suggestion
	cometbftrpc "github.com/cometbft/cometbft/rpc/client"
	cometbfthttp "github.com/cometbft/cometbft/rpc/client/http"
```

https://google.github.io/styleguide/go/decisions.html#package-names